src/assets/netteForms.js

```diff
@@ -27,17 +33,27 @@
 
 var Nette = {};
 
+function getHandler(callback) {
+	return function(evt) {
+		callback.call(this, evt || window.event);
```

Comment:
Couldn't this break https://github.com/jahudka/forms/blob/dca62a0029af4b558aa1c7572968fc274a5dc11d/src/assets/netteForms.js#L580 ?